src/game/AI/ScriptDevAI/scripts/kalimdor/dustwallow_marsh.cpp

```diff
@@ -1000,6 +1001,175 @@ UnitAI* GetAI_mob_invis_firework_helper(Creature* pCreature)
     return new mob_invis_firework_helper(pCreature);
 }
 
+/*######
+## npc_theramore_practicing_guard
+######*/
+enum
+{
+    NPC_THERAMORE_COMBAT_DUMMY = 4952,
+};
+
+struct npc_theramore_practicing_guardAI : public ScriptedAI
+{
+    npc_theramore_practicing_guardAI(Creature* pCreature) : ScriptedAI(pCreature)
+    {
+        attackableDummy = GetClosestCreatureWithEntry(m_creature, NPC_THERAMORE_COMBAT_DUMMY, 2.f);
```

Comment:
This can result in nullptr when the Guard spawns sooner than the Dummy.

Creatures do not spawn at once, but one after another.